### PR TITLE
Book list i

### PR DESCRIPTION
**Pull request by Juliocho990**
## Summary
- List view was added into [list.html](https://github.com/wolox-training/jlb-angular/blob/book-list-I/list.html).
- Flex box was used to manage the list.

## Screenshot
book-list-I

## Trello card
https://trello.com/c/v1rBjuPG/30-listado-de-libros

### CODE REVIEW

**Review by CrisVega08 — CHANGES_REQUESTED** [review]
Same

File: scss/styles.scss [hunk 1]
```diff
@@ -5,3 +5,12 @@
   width: 100%;
   height: 100%;
 }
+
+.card {
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  background: $white;
+  display: flex;
+  flex-flow: row wrap;
```
Comment (eledwinn) [inline scss/styles.scss]: Why card have flex wrap? it is necessary?

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/list.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    .card {
+      flex-flow: column nowrap;
+      margin: 18px;
+      max-height: 300px;
+      max-width: 198px;
+      padding: 28px 28px 23px 28px;
+    }
+
+    .cover {
```
Comment (CrisVega08) [inline scss/list.scss]: this is so much specificity so 

use classes in this way for easier overriding 

.list  {
  ...
}

.content {
  ...
}
.author {
  ...
}

.title {
  ...
}
etc

File: scss/list.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      font-weight: bold;
+      margin: 0 0 8px 0;
+    }
+
+    .card {
+      flex-flow: column nowrap;
+      margin: 18px;
+      max-height: 300px;
+      max-width: 198px;
+      padding: 28px 28px 23px 28px;
```
Comment (CrisVega08) [inline scss/list.scss]:
```suggestion
      padding: 28px 28px 23px;
```

File: scss/list.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    .author {
+      color: $gray;
+      font-size: 14px;
+    }
+    
+    .title {
+      font-size: 16px;
+      font-weight: bold;
+      margin: 0 0 8px 0;
```
Comment (CrisVega08) [inline scss/list.scss]:
```suggestion
      margin-bottom: 8px;
```

File: scss/list.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    .card {
+      flex-flow: column nowrap;
```
Comment (CrisVega08) [inline scss/list.scss]:
```suggestion
      flex-flow: column;
```

File: scss/styles.scss (hunk 1 above)
Comment (CrisVega08) [reply scss/styles.scss]: i think is unnecessary

File: scss/styles.scss [hunk 6]
```diff
@@ -5,3 +5,12 @@
   width: 100%;
   height: 100%;
 }
+
+.card {
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
```
Comment (CrisVega08) [inline scss/styles.scss]: Use a variable for rgba(0,0,0,0.5);

File: scss/list.scss [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+      margin: 18px;
+      max-height: 300px;
+      max-width: 198px;
+      padding: 28px 28px 23px 28px;
+    }
+
+    .cover {
+      margin: 0 0 12px 0;
```
Comment (CrisVega08) [inline scss/list.scss]: margin-bottom: 12px

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/styles.scss (hunk 1 above)
Comment (Juliocho990) [reply scss/styles.scss]: ok

File: scss/list.scss (hunk 2 above)
Comment (Juliocho990) [reply scss/list.scss]: ok

File: scss/list.scss (hunk 3 above)
Comment (Juliocho990) [reply scss/list.scss]: ok

File: scss/list.scss (hunk 4 above)
Comment (Juliocho990) [reply scss/list.scss]: ok

File: scss/list.scss (hunk 5 above)
Comment (Juliocho990) [reply scss/list.scss]: ok

File: scss/styles.scss (hunk 6 above)
Comment (Juliocho990) [reply scss/styles.scss]: ok

File: scss/list.scss (hunk 7 above)
Comment (Juliocho990) [reply scss/list.scss]: ok

**Review by CrisVega08 — APPROVED** [review]

**Review by eledwinn — APPROVED** [review]

**Pull request merged**